src/components/SingUpForm.tsx

```diff
@@ -0,0 +1,84 @@
+import React, { useState } from "react";
+import { Link } from "react-router-dom";
+import { toast } from "react-toastify";
+import { Button, Form, FormInput } from "semantic-ui-react";
+import { singUpRequest } from "../api";
+
+interface Props {
```

Comment:
interface for router already exists. No need to describe it

Reply:
fixed. I import interface from other form